### PR TITLE
[terra-toggle]fixing focus issue & added expanded & collapse state 

### PR DESCRIPTION
**Pull request by udaymattam**
### Summary

fixed focus issues & added label example for terra-toggle examples.

### Deployment Link

https://terra-core-deployed-pr-#.herokuapp.com/

### Additional Details

Thank you for contributing to Terra.
@cerner/terra

### CODE REVIEW

File: packages/terra-core-docs/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,11 @@
 
 ## Unreleased
 
+* Changed
+  * Fixed Focus Issue & Expand/collapse state for `terra-toggle` examples.
+* Added 
+  * Added Label Example for `terra-toggle` examples.
```
Comment (supreethmr) [inline packages/terra-core-docs/CHANGELOG.md:8]:
```suggestion
* Changed
  * Updated examples to use informative icons

* Added 
  * Added Label Example for `terra-toggle`.
```

File: packages/terra-core-docs/src/terra-dev-site/doc/toggle/example/ToggleWithLabel.jsx [hunk 2]
```diff
@@ -1,21 +1,37 @@
...
+  const cx = classNamesBind.bind(styles);
 
+  const toggleClasses = classNames(
+    cx({ 'is-focused': focused }),
+  );
   return (
     <div>
       <label htmlFor="Icon-label">
-        <IconInformation className={cx('is-focused')} onClick={handleOnClick} a11yLabel="information Icon" tabIndex="0" role="button" area-labelledby="Toggle label" aria-expanded={isOpen} aria-controls="toggle" id="Icon-label" />
+        <IconInformation className={toggleClasses} onClick={handleOnClick} handleKeyUp={handleKeyUp} handleKeyDown={handleKeyDown} a11yLabel="information Icon" tabIndex="0" role="button" area-labelledby="Toggle label" aria-expanded={isOpen} aria-controls="toggle" id="Icon-label" />
```
Comment (supreethmr) [inline packages/terra-core-docs/src/terra-dev-site/doc/toggle/example/ToggleWithLabel.jsx:34]: why do we need aria-labelledby here ?? ( typo area-labelledBy )

File: packages/terra-core-docs/src/terra-dev-site/doc/toggle/example/DefaultToggle.jsx [hunk 3]
```diff
@@ -1,17 +1,24 @@
...
+import { KEY_TAB } from 'keycode-js';
+import './ToggleExample.module.scss';
 
 const ToggleDefault = () => {
   const [isOpen, setIsOpen] = useState(false);
-
+  const [focused, setFocus] = useState(true);
   const handleOnClick = () => {
     setIsOpen(!isOpen);
+    setFocus(false);
```
Comment (supreethmr) [inline packages/terra-core-docs/src/terra-dev-site/doc/toggle/example/DefaultToggle.jsx:12]: here also you can remove this state update

File: packages/terra-core-docs/src/terra-dev-site/doc/toggle/example/ToggleWithLabel.jsx [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+import IconInformation from 'terra-icon/lib/icon/IconInformation';
+import { KEY_TAB } from 'keycode-js';
+import './ToggleExample.module.scss';
+
+const ToggleWithLabel = () => {
+  const [isOpen, setIsOpen] = useState(false);
+  const [focused, setFocus] = useState(true);
+  const handleOnClick = () => {
+    setIsOpen(!isOpen);
+    setFocus(false);
```
[the remark this answers is not in the document]
Comment (udaymattam) [reply packages/terra-core-docs/src/terra-dev-site/doc/toggle/example/ToggleWithLabel.jsx]: removed

**Pull request merged**